src/index.html

```diff
@@ -1,13 +1,171 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="top-bar">
+        <a  href="#" class="top-bar__logo-link">
+          <img
+            src="./images/Logo.png"
+            alt="BOSE logo"
+            class="top-bar__logo"
+          >
+        </a>
+
+        <div class="top-bar__icons">
+          <a href="#" class="icon icon--phone"></a>
+          <a href="#menu" class="icon icon--menu"></a>
+        </div>
+      </div>
+
+      <h1 class="header__title">Your senses.<br>Set free.</h1>
+    </header>
+
+    <aside class="menu page__menu" id="menu">
+      <div class="top-bar menu__top">
+        <a  href="#" class="top-bar__logo-link">
+          <img
+            src="./images/Logo.png"
+            alt="BOSE logo"
+            class="top-bar__logo"
+          >
+        </a>
+
+        <div class="header__icons">
+          <a href="#" class="icon icon--close"></a>
+        </div>
+      </div>
+
+      <div class="menu__content">
+        <nav class="nav menu__nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#recommended"
+                class="nav__link"
+              >
+                Recommended
+              </a>
+
+            </li>
+            <li class="nav__item">
+              <a
+                href="#categories"
+                class="nav__link"
+              >
+                Categories
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#about-us"
+                class="nav__link"
+              >
+                About us
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#contacts"
+                class="nav__link"
+              >
+                Contacts
+              </a>
+            </li>
+          </ul>
+        </nav>
+
+        <a href="tell:+1 234 5555-55-55" class="menu__phone-number">
```

Comment:
The correct format is tel:
```suggestion
        <a href="tel:+1 234 5555-55-55" class="menu__phone-number">
```